### DISCUSSION TITLE
Bots try to help husk

### DISCUSSION THREAD

**Discussion by Sakirogrono** [FakeFishGames/Barotrauma#15094] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

For some reason my bots try to heal other husk even though they dont have the husk symbiosis or genes even but they're wearing the zealot robes 

![image](https://github.com/user-attachments/assets/d920f735-f5e4-4578-a746-859d0469fae3)

### Reproduction steps

1. Make sure your crewmember is wearing a zealot robe
2. Make sure its a medic
3. Injure a husk or any types of it see if it tries to heal it

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Single player

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by A2ur3** [comment]
#14996

**Comment by itchyOwl** [comment]
Has been fixed in [Miracle Worker talent affecting husks #5755](https://github.com/FakeFishGames/Barotrauma-development/pull/5755) (included in the latest release)

Confirmed by cross-testing the issue in autumn-update and winter-update.
